tests/NLog.UnitTests/LayoutRenderers/Log4JXmlTests.cs

```diff
@@ -156,6 +156,10 @@ public void Log4JXmlTest()
                                         break;
 #endif
 
+                                    case "nlogPropertyKey":
```

Comment:
isn't it strange we have now "nlogPropertyKey" in "log4j" and "nlog" (https://github.com/snakefoot/NLog/blob/2f5e3cced2fb1e56846e68e260c28c7d237bccfb/tests/NLog.UnitTests/LayoutRenderers/Log4JXmlTests.cs#L192)